index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
 
   <title>My Resume</title>
-
+  <link href="https://fonts.googleapis.com/css?family=Noto+Serif" rel="stylesheet">
```

Comment:
good job bringing in the google fonts, but it looks like you're missing a certain meta tag that should be around this area. Let me know when you add it in.